smoke-tests/apps/DetectUnexpectedOtelMetrics/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/DetectUnexpectedOtelMetricsTest.java

```diff
@@ -23,14 +25,22 @@ abstract class DetectUnexpectedOtelMetricsTest {
 
   @RegisterExtension static final SmokeTestExtension testing = SmokeTestExtension.create();
 
+  private static final List<String> EXPECTED_METRIC_NAMES = new ArrayList<>();
+
+  static {
+    EXPECTED_METRIC_NAMES.add("_OTELRESOURCE_");
+    EXPECTED_METRIC_NAMES.add("Current Thread Count");
+    EXPECTED_METRIC_NAMES.add("Loaded Class Count");
```

Comment:
are these 2 needed? (wondering why only these and not other performance counters)

Reply:
perf counters are not otel metrics yet.